apisix/plugins/forward-auth.lua

```diff
@@ -78,6 +78,8 @@ local _M = {
 
 
 function _M.check_schema(conf)
+    local check = {"uri"}
```

Comment:
The `ssl_verify` should also be checked, TLS without server certificate validation is meaningless.